src/util/error.cpp

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2010-2018 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include <util/error.h>
+
+#include <util/system.h>
+
+std::string TransactionErrorString(const TransactionError err)
```

Comment:
This commit should be "moving" this, but we don't have it yet. Should maybe backport what adds this?

Reply:
Introduced in a series of changes brought in by https://github.com/bitcoin/bitcoin/pull/14978, cannot be merged as PSBT is not merged. Will remove line